src/main/scala/ignition/core/jobs/utils/SparkContextUtils.scala

```diff
@@ -26,6 +28,11 @@ object SparkContextUtils {
     override def getPartition(key: Any): Int = index(key)
   }
 
+  implicit class S3ObjectSummaryExtensions(s3Object: S3ObjectSummary) {
+    def toHadoopFile: HadoopFile =
+      HadoopFile(s"s3n://${s3Object.getBucketName}/${s3Object.getKey}", isDir = false, s3Object.getSize)
+  }
```

Comment:
I can not say that I like explicit classes. Specially for a single usage. But is me :P